components/buttons/buttons-alt.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <div class="row">
+    <div class="col-lg">
+      <div class="row mb-2">
+        <button type="button" class="btn btn-primary m-2">Primary</button>
+        <button type="button" class="btn btn-primary disabled m-2">Primary</button>
+        <button type="button" class="btn btn-outline-primary m-2">Primary</button>
+
+      </div>
+      <div class="row mb-2">
+        <button type="button" class="btn btn-secondary m-2">Secondary</button>
+        <button type="button" class="btn btn-secondary disabled m-2">Secondary</button>
+        <button type="button" class="btn btn-outline-secondary m-2">Secondary</button>
+      </div>
+      <div class="row mb-2">
+        <button type="button" class="btn btn-link m-2">Link</button>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Do we need this file if we also have the primary and secondary ones? Maybe we move the plain link to the `button-primary` file?

Reply:
Agreed. I had forgotten to remove this.

Reply:
No worries, I'll get this too.